### DISCUSSION TITLE
lotus-miner init  bug:   message execution failed: exit 16, reason: failed to init new actor

### DISCUSSION THREAD

**Discussion by colccool** [filecoin-project/lotus#6863] [Storage Provider]
when I run "lotus-miner init --worker  --no-local-storage --nosync"

lotus version 1.11.1-dev+calibnet+git.dd0966639

Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal

2021-07-23T18:25:26.677+0800	INFO	badger	v2@v2.2007.2/levels.go:183	All 0 tables opened in 0s

2021-07-23T18:25:26.703+0800	INFO	badger	v2@v2.2007.2/levels.go:183	All 0 tables opened in 0s

2021-07-23T18:25:26.723+0800	ERROR	main	lotus-storage-miner/init.go:261	Failed to initialize lotus-miner: creating miner failed:
    main.storageMinerInit
        /home/bitchina/lotus/cmd/lotus-storage-miner/init.go:541
  - pushing createMiner message:
    main.createStorageMiner
        /home/bitchina/lotus/cmd/lotus-storage-miner/init.go:713
  - GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: failed to init new actor (RetCode=16)
2021-07-23T18:25:26.723+0800	INFO	main	lotus-storage-miner/init.go:266	Cleaning up .lotusminer after attempt...
ERROR: Storage-miner init failed

**Comment by colccool** [comment]
I try much time ,but still not work

**Comment by jennijuju** [comment]
Do you have more error logs?

**Comment by colccool** [comment]
no,just this




------------------ Original ------------------
From: Jennifer ***@***.***&gt;
Date: Sun,Jul 25,2021 10:19 AM
To: filecoin-project/lotus ***@***.***&gt;
Cc: sexfody ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [filecoin-project/lotus] lotus-miner init  bug:   message execution failed: exit 16, reason: failed to init new actor (#6863)





 
Do you have more error logs?

**Comment by chop8082** [comment]
I have the same problem

**Reply by colccool** [reply]
Have you solved the problem

**Reply by chop8082** [reply]
no

**Reply by colccool** [reply]
the worker or owner must be bls address

**Comment by de-authority** [comment]
Do you have enough balance in your own address.

**Comment by LexLuthr** [comment]
@colccool Were you able to resolve this or do you still need help with it? If it was resolved, can you tell us what the problem was?
